src/fetch/dune.py

```diff
@@ -59,7 +59,7 @@ def _parameterized_query(
     def _get_query_results(self, query: Query) -> list[dict[str, str]]:
         """Internally every dune query execution is routed through here."""
         log.info(f"Fetching {query.name} from query: {query}")
-        exec_result = self.dune.refresh(query, ping_frequency=5)
+        exec_result = self.dune.refresh(query, ping_frequency=15)
```

Comment:
unrelated?

Reply:
Yep totally. I really thought I had commented somewhere about this. Essentially none of our query use take less than 15 seconds to execute anyway and this change would reduce our chances of being rated limited.